apps/wallet/src/ui/app/experimentation/feature-gating.ts

```diff
@@ -4,7 +4,14 @@
 import { GrowthBook } from '@growthbook/growthbook';
 
 import type { JSONValue } from '@growthbook/growthbook';
-import type { WidenPrimitives } from '@growthbook/growthbook/dist/types/growthbook';
+
+type WidenPrimitives<T> = T extends string
```

Comment:
Rather than doing this, I'd just change the definition of the function.

If you want to keep the explicit return:
```ts
public getFeatureValue<T extends JSONValue>(
        featureName: string,
        defaultValue: T
): ReturnType<GrowthBook['getFeatureValue']> {
```

But you can also use inferred types here which I think is fine: 

```ts
public getFeatureValue<T extends JSONValue>(
        featureName: string,
        defaultValue: T
) {
```

You also could define the entire function to satisfy the same type:
```ts
public getFeatureValue: GrowthBook['getFeatureValue'] = (
        featureName,
        defaultValue
) => {
```